### PR TITLE
API documentation for alert catalogue

### PR DESCRIPTION
**Pull request by navish**
Ref #16

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -2952,7 +2952,386 @@ HTTP/1.1 200 Success
 
 ## EventCatalogue
 
-> TODO
+`EventCatalogue` maps an emergency response activity to the responsible [party](#core-resources-party), [role](#core-resources-partyrole) or [group](#core-resources-partygroup)
```
Comment (rijkerd) [inline docs/api.md]: Party, Role, Group should be in caps. eg PARTY, ROLE, GROUP

File: docs/api.md [hunk 2]
```diff
@@ -2952,7 +2952,386 @@ HTTP/1.1 200 Success
...
+|                  updatedAt                  | String | A time value given in ISO8601 combined date and time format that represents when the catalogue was updated. |
+
+> Example Response
+
+```curl
+HTTP/1.1 200 Success
+{
+"_id": "5de7b7bb9a6ed7342b200f02",
+"type": {
+  "_id": "5de7b7bb9a6ed7342b200f02"
```
Comment (rijkerd) [inline docs/api.md]: Missing comma (,)

File: docs/api.md [hunk 3]
```diff
@@ -2952,7 +2952,386 @@ HTTP/1.1 200 Success
...
+},
+],
+"roles": [{
+  "_id": "5de7b7b79a6ed7342b200ddc",
+  "name": { "en": "Regional Admnistrative Secretary" },
+}],
+"agencies": [],
+"focals": [],
+"updatedAt": "2019-12-16T13:42:46.937Z",
+"createdAt": "2019-12-16T13:42:20.186Z",
```
Comment (rijkerd) [inline docs/api.md:3268]: Missing deletedAt and add on attributes

File: docs/api.md [hunk 4]
```diff
@@ -2952,7 +2952,386 @@ HTTP/1.1 200 Success
...
+}
```
Comment (rijkerd) [inline docs/api.md]: Missing attributes check event docs example

**Review by rijkerd — CHANGES_REQUESTED** [review]
Check on the data type ie type that should be an object

File: docs/api.md (hunk 1 above)
Comment (navish) [reply docs/api.md]: Done

**Review by rijkerd — APPROVED** [review]

**Review by lykmapipo — APPROVED** [review]

**Pull request merged**